recipes-bsp/atf/atf-poplar_git.bb

```diff
@@ -0,0 +1,41 @@
+DESCRIPTION = "ARM Trusted Firmware Poplar"
```

Comment:
ack, as discussed yesterday at our sync, let's keep it that way for now. We'll worry about it when we get UEFI support. Providing support for both available bootloader is nice but from an automated testing point of view (e.g. in LAVA) it could be problematic. As you said, we don't want to test the bootloader but the OP-TEE features provided.